app/components/search-facet-worktype/style.scss

```diff
@@ -1,3 +1,17 @@
 .row {
-    text-align: center;
+  display: flex;
+  flex-direction: column;
+
+  > button {
```

Comment:
You may want to specify some selectors (hover, active, target) to make it clear which button is currently clicked.